cc/main.cc

```diff
@@ -145,6 +151,17 @@ std::string GetOutputName(absl::Time now, size_t i) {
   return absl::StrCat(timestamp, "-", hostname, "-", i);
 }
 
+std::string GetRowPrefix(absl::Time now, int id) {
+  std::ostringstream out;
+  absl::TimeZone utc = absl::UTCTimeZone();
+  out << absl::FormatTime("%Y%m%dT%H%M%S", now, utc)
```

Comment:
This the T required? could it be changed to a _?

Reply:
I used the `T` only because that's ISO 8601, but honestly, there's no reason why it has to conform to that. The valuable part is having a timestamp that sorts lexicographically. I'll change it.